org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/IFilterOutput.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2017 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.analysis.filter;
+
+import org.objectweb.asm.tree.AbstractInsnNode;
+
+public interface IFilterOutput {
+
+	void ignore(AbstractInsnNode from, AbstractInsnNode to);
```

Comment:
@marchof Actually both are inclusive. And we already use term "successor" in analysis of CFG, where this usage implies that not each instruction is a "successor" of preceding instruction from a list :wink: so not sure that "successor" will be a good unambiguous word here.